packages/select/src/components/query-list/queryList.tsx

```diff
@@ -417,11 +423,18 @@ export class QueryList<T> extends AbstractComponent2<QueryListProps<T>, IQueryLi
     private getActiveElement() {
         const { activeItem } = this.state;
         if (this.itemsParentRef != null) {
+            const { getActiveElement } = this.props;
             if (isCreateNewItem(activeItem)) {
                 const index = this.isCreateItemFirst() ? 0 : this.state.filteredItems.length;
+                if (typeof getActiveElement === "function") {
+                    return getActiveElement({ activeItem, index, itemsParent: this.itemsParentRef });
```

Comment:
Passing all the props that might be useful when implementing a custom `getActiveElement`